dashboard/pages/primary_screening/stages/s4_echo_input.py

```diff
@@ -1,5 +1,16 @@
 from dash import html, dcc
 
+ECHO_DESC = """
```

Comment:
I would consider providing a short description instead of `Lorem ipsum` as we want to show the progress to the client (same in the case of  `stage 1`)